httprange.go

```diff
@@ -36,93 +39,77 @@ func (e InvalidRange) Error() string {
 
 // HttpRange specifies the byte range to be sent to the client.
 type httpRange struct {
```

Comment:
BytePos is for the protocol.
By native I meant, for us, once we do parseRequestRange() to get httpRange we never use lastBytePos and we have to call getLength() to get the length. The term byte position will not hold meaning for us, it will be offset and length.